### ISSUE TITLE
Emit current value right on subscription to observable

### ISSUE THREAD

**Issue by semanticer** [pwittchen/prefser#33]
When using this library, common use case is when I want to get the current value from prefser and populate my UI with it and then subscribe for change and update my UI again with any new value that comes from onNext. I don't know... maybe it would be better to emit current value right when I subscribe so we can skip the first step and get rid off the getter. Just fluently bind prefser to my views, using getter first seems too... imperative :) 
I think this way, it would be more similar to approach of sqlbrite for example.

**Comment by pwittchen** [comment]
Nice idea! I'll take a closer look on that, so we'll be able to enhance current solution.

**Comment by pwittchen** [comment]
Emitting value from preferences in `observe(...)` method may be misleading for some developers. Moreover, someone may way want to get data without `Observable`. Nevertheless, your issue sounds reasonable, that's why I've added additional method called `getAndObserve(...)`. You can subscribe this method in exactly the same way as observe(...) method. The only difference is the fact that this method will emit value from SharedPreferences as first element of the stream with get(...) method even if SharedPreferences were not changed. When SharedPreferences changes, subscriber will be notified about the change in the same way as in regular observe(...) method.

Basically, this method should satisfy your needs.
You can read more in documentation at: https://github.com/pwittchen/prefser#getandobserve-method

This feature will be available in the next version of the library, which will be released this week.

**Comment by semanticer** [comment]
This is a great solution, thank you!
